app/views/RoomView/Header/LeftButtons.js

```diff
@@ -32,30 +32,22 @@ const LeftButtons = React.memo(({
 		);
 	}
 	const onPress = useCallback(() => goRoomActionsView(), []);
-	if (baseUrl && userId && token) {
-		return (
-			<Avatar
-				text={title}
-				size={30}
-				type={t}
-				baseUrl={baseUrl}
-				style={styles.avatar}
-				userId={userId}
-				token={token}
-				onPress={onPress}
-			/>
-		);
-	}
-	return null;
+
+	return (
```

Comment:
IMO we should keep returning `null` here instead of relying to `Avatar` the responsibility